stories/atoms/components/typography/Typography.stories.mdx

```diff
@@ -137,29 +143,82 @@ There is no default size to preserve what's being ultimately rendered.
-        </Typography>
-        <Typography as="div" color="success" fontFamily="brand" fontSize="medium">
-          success - OKP4 - At the core of the stakes of data valorisation
-        </Typography>
-        <Typography as="div" color="warning" fontFamily="brand" fontSize="medium">
-          warning - OKP4 - At the core of the stakes of data valorisation
-        </Typography>
-        <Typography as="div" color="error" fontFamily="brand" fontSize="medium">
-          error - OKP4 - At the core of the stakes of data valorisation
-        </Typography>
-      </div>
+  The basic colors.
+  <Story name="Basic colors">
+    {() => (
+      <>
+        <div
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            padding: '20px',
+            gap: '20px'
+          }}
+        >
+          <Typography as="div" color="text" fontFamily="brand" fontSize="medium">
+            text | OKP4 - At the core of the stakes of data valorisation
+          </Typography>
+          <Typography as="div" color="highlighted-text" fontFamily="brand" fontSize="medium">
+            highlighted-text | OKP4 - At the core of the stakes of data valorisation
+          </Typography>
+          <Typography as="div" color="info" fontFamily="brand" fontSize="medium">
+            info | OKP4 - At the core of the stakes of data valorisation
+          </Typography>
+          <Typography as="div" color="success" fontFamily="brand" fontSize="medium">
+            success | OKP4 - At the core of the stakes of data valorisation
+          </Typography>
+          <Typography as="div" color="warning" fontFamily="brand" fontSize="medium">
+            warning | OKP4 - At the core of the stakes of data valorisation
+          </Typography>
+          <Typography as="div" color="error" fontFamily="brand" fontSize="medium">
+            error | OKP4 - At the core of the stakes of data valorisation
+          </Typography>
+        </div>
+      </>
+    )}
+  </Story>
+  The specific cases colors.
+  <Story name="Specific cases colors">
+    {() => (
+      <>
+        <div
+          className="high-contrast-component-background"
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            padding: '20px',
+            gap: '20px'
+          }}
+        >
+          <Typography as="div" color="inverted-text" fontFamily="brand" fontSize="medium">
+            inverted-text | OKP4 - At the core of the stakes of data valorisation
+          </Typography>
+          <Typography as="div" color="primary-invariant" fontFamily="brand" fontSize="medium">
+            primary-invariant | OKP4 - At the core of the stakes of data valorisation
+          </Typography>
+          <Typography as="div" color="secondary-invariant" fontFamily="brand" fontSize="medium">
+            secondary-invariant | OKP4 - At the core of the stakes of data valorisation
+          </Typography>
```

Comment:
Great, but the `secondary-invariant` is almost unreadable in dark theme, ne weed to find a way to fix that:

![image](https://user-images.githubusercontent.com/47554752/187226738-2157c57c-5a67-4cdb-a6e4-4a2efa81e2c1.png)